collectors/auth0/auth0_collector.js

```diff
@@ -76,7 +69,15 @@ class Auth0Collector extends PawsCollector {
         const nextPollInterval = nowMoment.diff(lastLogMoment, 'seconds') > this.pollInterval ?
                 1 : this.pollInterval;
 
-        return  {
+        if (nextLogId === null) {
+            //If collector initial call and get empty logs then this condition will work
+            return {
+                since: curState.since,
+                poll_interval_sec: 1
+            };
+        }
+
+        return {
             last_log_id: nextLogId,
```

Comment:
Could you add a `last_collected_ts` key to this state and make it equal to the `lastLogTs`? We need this for metrics reporting.